### PR TITLE
Cleanup kotlin sources

### PR DESCRIPTION
**Pull request by quaff**
1. remove unused imports
2. remove redundant semicolon
3. remove empty class body
4. remove redundant 'constructor' keyword
5. remove redundant 'Unit' return type
6. use non-null type if possible

### CODE REVIEW

File: spring-boot-project/spring-boot-docs/src/main/kotlin/org/springframework/boot/docs/actuator/metrics/supported/mongodb/command/MyCommandTagsProviderConfiguration.kt [hunk 1]
```diff
@@ -24,8 +24,6 @@ import org.springframework.context.annotation.Configuration
 class MyCommandTagsProviderConfiguration {
 
 	@Bean
-	fun customCommandTagsProvider(): MongoCommandTagsProvider? {
-		return CustomCommandTagsProvider()
-	}
+	fun customCommandTagsProvider() = CustomCommandTagsProvider()
```
Comment (mhalbritter) [inline spring-boot-project/spring-boot-docs/src/main/kotlin/org/springframework/boot/docs/actuator/metrics/supported/mongodb/command/MyCommandTagsProviderConfiguration.kt:27]: I'm not a fan of this change, as this removes the explicit `MongoCommandTagsProvider` return type.

**Review by mhalbritter — COMMENTED** [review]
Hey, I left a comment. Besides the one change this looks good to me.

File: spring-boot-project/spring-boot-docs/src/main/kotlin/org/springframework/boot/docs/actuator/metrics/supported/mongodb/command/MyCommandTagsProviderConfiguration.kt (hunk 1 above)
Comment (quaff) [reply spring-boot-project/spring-boot-docs/src/main/kotlin/org/springframework/boot/docs/actuator/metrics/supported/mongodb/command/MyCommandTagsProviderConfiguration.kt]: done.

**Comment by mhalbritter** [comment]
Thank you!

**Pull request closed without merging**